test/into.js

```diff
@@ -29,6 +29,12 @@ describe('into', function() {
         assert.deepEqual(R.into({}, R.identity, [{a: 1}, {b: 2, c: 3}]), {a: 1, b: 2, c: 3});
     });
 
+    it('dispatches to objects that implement `reduce`', function() {
+        var obj = {x: [1, 2, 3], reduce: function(f, acc) { return R.reduce(f, acc, this.x); }};
+        assert.deepEqual(R.into([], R.map(add(1)), obj), [2, 3, 4]);
+        assert.deepEqual(R.into([], R.map(add(1)), obj), [2, 3, 4]);
+    });
+
```

Comment:
Do you mean adding a `reduceCalled` flag in the `obj.reduce` method or using a different `reduce` or something else?  The method itself must be called here, (how else would it get access to the list `[1, 2, 3]`?)